src/bmp_reader.cpp

```diff
@@ -1,31 +1,25 @@
 #include "bmp_reader.h"
 #include <iostream>
 #include <algorithm>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
```

Comment:
Why did you add this three libraries?

Reply:
Como estaba usando Replit y me estaba presentando error el string y llame la librería en todos los archivos en h y cpp. Aunque en el código h  no tenga string

 cstdio y  cstdlib
Las utilise porque el editor me la colocó como sugerencia y las agregue.

Reply:
Como estaba usando Replit y me estaba presentando error el string y llame la librería en todos los archivos en h y cpp. Aunque en el código h  no tenga string 

 cstdio y  cstdlib
Las utilise porque el editor me la colocó como sugerencia y las agregue.